rotkehlchen/tests/conftest.py

```diff
@@ -138,3 +140,44 @@ def requires_env(allowed_envs: list[TestEnvironment]):
         'CI' in os.environ and env not in allowed_envs,
         reason=f'Not suitable envrionment {env} for current test',
     )
+
+
+def get_cassette_dir(request: pytest.FixtureRequest) -> str:
```

Comment:
I would say that is better to return a `Path` object here and in `vcr_cassette_dir`

Reply:
`vcr_cassette_dir` is used by `pytest-recording` which expects `str`, but I'm happy to change it for this function